package.json

```diff
@@ -16,5 +16,9 @@
   "version": "0.1.0",
   "devDependencies": {
     "webpack-dev-server": "^3.4.1"
+  },
+  "resolutions": {
+    "minimist": "^1.2.5",
```

Comment:
mkdirp is enough to trigger the upgrade of minimist, since it is the new version of mkdirp requires a more up to date minimist.
Actually, the resolutions part of package.json can be removed after yarn has upgraded the yarn.lock.

But I haven't got a lot of experience with yarn.lock. So I am unsure about how a package.json preferably looks.